.github/workflows/metrics_analysis.yml

```diff
@@ -9,6 +9,7 @@ jobs:
   metrics-analysis:
     runs-on: ubuntu-latest
     permissions:
+      contents: read
```

Comment:
We should have added this to say that this job is allowed to read contents from the git repo. The fact that we didn't wasn't causing us trouble (perhaps because the repo is public), but this is good practice.